_data/faqs.yaml

```diff
@@ -1,76 +1,76 @@
 - question: >-
-    Is the Hydra-in-a-Box application dependent on Amazon Web Services? What if I want to host it on a different cloud service provider, or on local infrastructure?
-  answer: >-
-    No, the application is not at all tied to AWS. If you choose to run it on AWS, take advantage of the CloudFormation templates our project team developed (<a href="https://github.com/hybox/aws">available on GitHub</a>). If you choose to install it on another cloud service provider, you may need to do similar development for that environment. For local installations, you need not pay attention to any of this template business.
+    What is the relationship between Hyku, Hyrax, and Sufia? 
+  answer: >- 
+    Hyku is based directly on Hyrax. Hyku had been based on Sufia until November 2016 when Sufia and CurationConcerns were consolidated into one gem named Hyrax. From Hyku's perspective, there has been no functional change since Sufia was consolidated into Hyrax.
 - question: >-
-    I am interested in starting to use Hydra. If I start now, will I be able to migrate to Hydra-in-a-Box when it is ready? Or should I hold off and wait?
+    I am interested in starting to use Hydra at my institution. If I start now, will I be able to migrate to Hyku when it is released? Or should I hold off and wait?
   answer: >-
-    The Hydra-in-a-Box repository product is built using Sufia -- a Hydra community-managed software component -- as a starting point. If you are ready to start with Hydra today with the intent or possibility of moving to Hydra-in-a-Box, then you should start with Sufia. There will be a clear migration path to the Hydra-in-a-Box application from Sufia. <a href="http://sufia.io/">Read more about Sufia here</a>.<br/><br/>If you run a Sufia-based application that has local customizations, migration to Hydra-in-a-Box will likely be more complicated. The more customization, the more complex your migration will be. 
+    The Hyku repository product is built using Hyrax as its starting point. Hyrax is a consolidation of Sufia and CurationConcerns, two widely-adopted Hydra community-managed software components. If you are ready to start with Hydra today with the intent or potential of moving to Hyku, then you should start with Hyrax. There will be a clear migration path to Hyku from Hyrax- and Sufia-based applications. <a href="http://sufia.io/">Read more about Sufia here</a>.<br/><br/>If you run a Sufia- or Hyrax-based application that has local customizations, migration to Hyku will likely be more complicated. The more customization you have undertaken, the more complex your migration will be. 
```

Comment:
I'd be interested to get feedback from @jcoyne here re: what the upgrade path would look like from a gem (like Sufia or Hyrax) to a Rails application. I suspect it's more of a migration than an upgrade.